extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -213,6 +229,12 @@ export class ConsentUI {
         classList.add('amp-hidden');
       }
 
+      if (this.overlayEnabled_) {
```

Comment:
Have one code style idea, I think it might be a bit cleaner, let me know if you feel the same way. 

call `maybeHideOverlay_()` or `maybeShowOverlay_` (open to naming), and have the check to `this.overlayEnabled_`, within the helper method.